pkg/cmd/build/build.go

```diff
@@ -283,6 +283,23 @@ func OptsFromBuildInfo(manifestName, svcName string, b *model.BuildInfo, o *type
 		file = extractFromContextAndDockerfile(b.Context, b.Dockerfile, svcName)
 	}
 
+	args := []model.BuildArg{}
+	optionsBuildArgs := map[string]string{}
+	for _, arg := range o.BuildArgs {
+		splittedArg := strings.SplitN(strings.Trim(arg, "\""), "=", 2)
```

Comment:
in which cases do we need to `trim` the arg? Tested with `"MY_BUILD_ARG=ubuntu" seems to catch it without that func call.

Reply:
without `trim()` the `arg` and executing `ok build --progress plain --no-cache --build-arg "MY_BUILD_ARG=ubuntu"` I'm able to build the image

Reply:
using the following files:

````

ARG MY_BUILD_ARG=
FROM $MY_BUILD_ARG
````

````

build:
  web:
    context: .
    args:
      MY_BUILD_ARG: fdgdgsg
````

Reply:
I faced it using the debugger so  after checking it's working without it I'll go ahead and remove the trim. Thanks for spotting it